content/ko/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 클라우드 네이티브 보안 개요
+content_template: templates/concept
+weight: 1
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+쿠버네티스 보안(일반적인 보안)은 관련된 많은 부분이 상호작용하는 
+방대한 주제다. 오늘날에는 웹 애플리케이션의 실행을 돕는 
+수많은 시스템에 오픈소스 소프트웨어가 통합되어 있으며, 
+전체적인 보안에 대하여 생각할 수 있는 방법에 대한 통찰력을 도울 수 있는 
+몇가지 중요한 개념이 있다. 이 가이드는 클라우드 네이티브 보안과 관련된 
+몇가지 일반적인 개념에 대한 멘탈 모델을 정의한다. 멘탈 모델은 완전히 임의적이며 
+소프트웨어 스택을 보호할 위치를 생각하는데 도움이되는 경우에만 사용해야 
+한다.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 클라우드 네이티브 보안의 4C
+계층적인 보안에 대해서 어떻게 생각할 수 있는지 이해하는데 도움이 될 수 있는 다이어그램 부터 살펴보자.
+{{< note >}}
+이 계층화된 접근 방식은 보안에 대한 [심층 방어](https://en.wikipedia.org/wiki/Defense_in_depth_(computing)) 
```

Comment:
```suggestion
이 계층화된 접근 방식은 보안에 대한 [심층 방어](https://ko.wikipedia.org/wiki/%EC%A2%85%EC%8B%AC%EB%B0%A9%EC%96%B4)
```
?

Reply:
That link(ko.) is about military strategy of category, not about computing.
There are no Korean translated page about "defense in depth" with computing.

Reply:
If there's no matching page in Korean wikipedia, how about leaving the original link? Different content may cause misunderstanding.

Reply:
@gochist
Yes, left the original link.